src/hooks/useSafeNotifications.ts

```diff
@@ -26,12 +27,14 @@ const useSafeNotifications = (): void => {
   const { safe } = useSafeInfo()
   const { chainId, version, implementationVersionState } = safe
   const isOwner = useIsSafeOwner()
+  const safeAddress = useSafeAddress()
 
   /**
    * Show a notification when the Safe version is out of date
    */
 
   useEffect(() => {
+    if (safe.address.value !== safeAddress) return
```

Comment:
I think you can achieve the same like this:
```
const { safe, safeAddress } = useSafeInfo()

if (safeAddress) return
```

`safeAddress` from `useSafeInfo` becomes undefined when a Safe is being (re)loaded.

Reply:
Checking it! Thanks

Reply:
It turns undefined indeed when fetching but in the moment we call `useSafeNotifications` it is still returning the old safe address.

I found comparing the URL address with the loaded Safe address more reliable. WDYT?

Reply:
I see, thanks for checking!